flink-staging/flink-ml/src/main/scala/org/apache/flink/ml/classification/SVM.scala

```diff
@@ -187,6 +196,26 @@ class SVM extends Predictor[SVM] {
     parameters.add(Seed, seed)
     this
   }
+
+  /** Sets the threshold above which elements are classified as positive
+    *
+    * @param threshold
+    * @return
+    */
+  def setThreshold(threshold: Double): SVM = {
+    parameters.add(Threshold, threshold)
+    this
+  }
+
+  /** Clears the classification threshold, predictions made after calling this function will have
+    * the raw decision function value.
+    *
+    * @return
+    */
+  def clearThreshold(): SVM = {
+    parameters.add(Threshold, Option.empty[Double])
```

Comment:
I'm surprised that this compiles because you should only be able to add a `Double` here.